### ISSUE TITLE
Graphql request visible in devtools?

### ISSUE THREAD

**Issue by NickyMeuleman** [javascript-af/javascript-af#88]
Can't seem to see any graphql queries in the devtools network tab.
Tested on a couple different browsers. 
![Chrome](https://user-images.githubusercontent.com/30179461/52598523-428b4780-2e56-11e9-81e2-5237008a994d.gif)
![Firefox](https://user-images.githubusercontent.com/30179461/52598528-48812880-2e56-11e9-8f24-87160d457ee6.gif)
![Edge](https://user-images.githubusercontent.com/30179461/52598533-50d96380-2e56-11e9-84f3-135599968c33.gif)

Do see them on other sites that use graphql, like https://iamsaravieira.com/
![2019-02-11_23-41-08](https://user-images.githubusercontent.com/30179461/52598633-add51980-2e56-11e9-9a3a-bf17dfb747e0.gif)

Please tell me I'm being stupid and forgot some supersimple thing 🙏

**Comment by pantharshit00** [comment]
I don't think we make a query yet :)

**Comment by pantharshit00** [comment]
You called the refetch wrong the wrong way

**Comment by NickyMeuleman** [comment]
Thanks!
Glad I made a mistake and it's working as it should.
Not at pc with dev-things installed, will check tomorrow.

https://www.apollographql.com/docs/react/essentials/queries.html#refetching
> refetch takes variables, but if we don’t pass in new variables, it will use the same ones from our previous query.

Just to confirm my understanding:
Because I passed `onClick={payload.refetch}` it treated that call as a refetch with the same variables. Because there are none (so it's the same), Apollo grabbed the result from the cache and I never saw that graphql request in the console.

In the new code `refetch` is being called with an empty object (which is different) and apollo does fire the graphql request.
